unity/CITools/Unity.Cookbook/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using BuildDriver;
+using NiceIO;
+using RecipeEngine;
+using RecipeEngine.Api.Logging;
+using RecipeEngine.Platforms.Loaders;
+using Unity.Cookbook.Platforms;
+
+namespace Unity.Cookbook
+{
+    public static class Program
+    {
+        public static async Task<int> Main(string[] args)
+        {
+            var repoRoot = Paths.RepoRoot;
+            Directory.SetCurrentDirectory(repoRoot);
+
+            // GlobalSettings.Initialize(repoRoot);
+
+            // Delete the existing .yml files.  RecipeEngine does not delete old files.  Therefore deleting the files is necessary
+            // in order to handle renaming of a recipe or deleting of a recipe.
+            foreach (var file in repoRoot.Combine(".yamato").Files("*.yml"))
+                file.Delete();
```

Comment:
This might be functionality we could PR into NiceIO.

Reply:
I could have done
```
repoRoot.Combine(".yamato").Files("*.yml").Delete();
```

Is that what you mean?

Reply:
ah yeah there you go. I thought there was a way to do it through NiceIO but wasn't sure.